### DISCUSSION TITLE
How much ram will be used when load vlws from sdcard?

### DISCUSSION THREAD

**Discussion by ham0nd** [Bodmer/TFT_eSPI#2273] [Q&A - Fonts]
I want to display all chinese words in my project(a bluetooth music player) with ESP32,but I dont know whether it have enough RAM.
How much ram will be used when load vlws from sdcard? thanks ^_^

**Comment by ham0nd** [comment]
@Bodmer

**Comment by Bodmer** [comment]
12 bytes (glyph metrics) for each glyph loaded. Plus a small amount for parameters.

**Reply by ham0nd** [reply]
thanks.
